src/main/java/com/epam/training/auction/WebSecurityConfig.java

```
@@ -27,7 +27,15 @@ protected void configure(HttpSecurity http) throws Exception {
     @Autowired
     public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {

```

Throw a custom Exception instead of a generic one and it is better to mask this exception in an unchecked exception instead of adding throws throws clause to the signature of the method.